BenchmarkDotNet.Artifacts/results/Polly.Core.Benchmarks.CircuitBreakerOpenedBenchmark-report-github.md

```diff
@@ -9,8 +9,8 @@ Job=MediumRun  Toolchain=InProcessEmitToolchain  IterationCount=15
 LaunchCount=2  WarmupCount=10  
 
 ```
-|                    Method |        Mean |     Error |    StdDev |      Median | Ratio | RatioSD |   Gen0 | Allocated | Alloc Ratio |
-|-------------------------- |------------:|----------:|----------:|------------:|------:|--------:|-------:|----------:|------------:|
-| ExecuteAsync_Exception_V7 | 29,720.5 ns | 177.63 ns | 265.86 ns | 29,629.0 ns | 56.63 |    0.58 | 0.0610 |    2888 B |       15.04 |
-| ExecuteAsync_Exception_V8 | 20,840.2 ns | 191.44 ns | 274.56 ns | 20,696.3 ns | 39.71 |    0.51 | 0.0610 |    1816 B |        9.46 |
-|   ExecuteAsync_Outcome_V8 |    525.1 ns |   1.07 ns |   1.47 ns |    525.0 ns |  1.00 |    0.00 | 0.0076 |     192 B |        1.00 |
+|                    Method |         Mean |     Error |    StdDev |  Ratio | RatioSD |   Gen0 | Allocated | Alloc Ratio |
+|-------------------------- |-------------:|----------:|----------:|-------:|--------:|-------:|----------:|------------:|
+| ExecuteAsync_Exception_V7 | 125,329.4 ns | 167.90 ns | 251.31 ns | 225.33 |    0.72 |      - |    2888 B |       15.04 |
```

Comment:
These ones are pretty big changes too.

Could you spin up a VM in Azure or something so the baseline is a bit more stable and run there?

Reply:
These are the numbers I get on my laptop:

![image](https://github.com/App-vNext/Polly/assets/103487740/b7a9620e-adc2-42af-90b5-0f458dcd3128)

Maybe I can just update those? We are mostly interested in allocations anyway. It's strange though, something must have happened on my devbox, I don't know the reason why it's reporting these inflated numbers.

Reply:
Uploaded laptop-based ones.